elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/example/PlayerStats.java

```diff
@@ -33,7 +33,7 @@
  */
 @Include
 @VersionQuery(sql = "SELECT COUNT(*) from playerStats")
-@EqualsAndHashCode
+@EqualsAndHashCode(callSuper = false)
```

Comment:
Using `true`, fails the test cases. So setting to `false` which is equivalent to default `skip`.